### DISCUSSION TITLE
blog/2024-01-05-caddy-is-better

### DISCUSSION THREAD

**Discussion by giscus** [crazychenz/vinnie.work#40] [vinnie.work Comments]
# blog/2024-01-05-caddy-is-better

Overview

https://www.vinnie.work/blog/2024-01-05-caddy-is-better

**Comment by AlexMikhalev** [comment]
For s3 browser check out rclone - you can use rclone mount to mount s3 like local filesystem and then proxy requests from caddy to rclone or rclone serve s3 to server s3 bucket over http. Rclone have a lot of options, worth having as part of your toolkit. 
Thank you for writing this up - it will save me time of not going by the same route.
